assets/pages/tracking/WatchlistPage.tsx

```diff
@@ -32,15 +33,23 @@ export default function WatchlistPage() {
     const onCreateWatchlist = (values: {
         name?: string
         domains: string[],
-        emailTriggers: string[]
-        connector?: string
+        triggers: string[]
+        connector?: string,
+        dsn?: string[]
     }) => {
         const domainsURI = values.domains.map(d => '/api/domains/' + d.toLowerCase())
+        let triggers = values.triggers.map(t => ({event: t, action: 'email'}))
+        if (values.dsn !== undefined) triggers = [...triggers, ...values.triggers.map(t => ({
```

Comment:
Space before and after blocks

```suggestion
        if (values.dsn !== undefined) {
                triggers = [...triggers, ...values.triggers.map(t => ({
```